core/src/main/resources/error/error-classes.json

```
@@ -1659,6 +1659,11 @@
       "Not allowed to implement multiple UDF interfaces, UDF class <className>."
     ]
   },
+  "NAMED_ARGUMENTS_SUPPORT_DISABLED" : {
+    "message" : [
+      "Cannot call function <functionName> because named argument references are not enabled here. In this case, the named argument reference was <argument>. Set spark.sql.allowNamedFunctionArguments to true to turn on feature."
```

Could you quote the config name and its value at the functions `toSQLConf()`/`toSQLConfVal` do.
```suggestion
      "Cannot call function <functionName> because named argument references are not enabled here. In this case, the named argument reference was <argument>. Set \"spark.sql.allowNamedFunctionArguments\" to \"true\" to turn on feature."
```

---

@MaxGekk I think @anchovYu said that it is preferred not to add the quotes. She was one who implemented the error-classes framework, and she said it was standard practice to call toSQLId() instead of adding the quotes in the error string.

---

ok, let's pass the config name from the code and quote it by `toSQLConf`. This works to me too.

---

Oh, sorry, I just realized that you meant it for functionName. This is good. I will commit this.